src/plugins/controls/common/control_group/control_group_panel_diff_system.ts

```diff
@@ -34,11 +34,11 @@ export const ControlPanelDiffSystems: {
         return false;
       }
 
-      const { value: valueA = ['', ''] }: Partial<RangeSliderEmbeddableInput> =
+      const { value: valueA = ['', ''], ...inputA }: Partial<RangeSliderEmbeddableInput> =
         initialInput.explicitInput;
-      const { value: valueB = ['', ''] }: Partial<RangeSliderEmbeddableInput> =
+      const { value: valueB = ['', ''], ...inputB }: Partial<RangeSliderEmbeddableInput> =
         newInput.explicitInput;
-      return isEqual(valueA, valueB);
+      return isEqual(valueA, valueB) && deepEqual(inputA, inputB);
```

Comment:
[This test](https://github.com/elastic/kibana/blob/main/test/functional/apps/dashboard_elements/controls/common/range_slider.ts#L127-L149) will catch this when https://github.com/elastic/kibana/pull/174201 is merged.